model/Lite/Lite.md

```diff
@@ -26,19 +26,61 @@ An SPDX Lite document can be used in parallel with SPDX documents in software su
 For a /Software/Package to be conformant with this profile,
 the following has to hold:
 
+1. The mincount for /Core/Element/spdxId is 1
 1. The mincount for /Core/Element/name is 1
+1. The mincount for /Core/Element/creationInfo is 1
+1. The mincount for /Core/Artifact/suppliedBy is 1
 1. The mincount for /Software/Package/packageVersion is 1
-1. The mincount for /Software/Package/packageUrl is 1
 1. The mincount for /Software/SoftwareArtifact/copyrightText is 1
 
-For a /Software/Sbom to be conformant with this profile,
-the following has to hold:
-
 1. for every `/Software/Package` there MUST exist exactly one `/Core/Relationship`
    of type `concludedLicense` having that element as its `from` property
    and an `/SimpleLicensing/AnyLicenseInfo` as its `to` property.
 1. for every `/Software/Package` there MUST exist exactly one `/Core/Relationship`
    of type `declaredLicense` having that element as its `from` property
    and an `/SimpleLicensing/AnyLicenseInfo` as its `to` property.
 
-- TODO - add other class restrictions
+For a /Software/Sbom to be conformant with this profile,
+the following has to hold:
+
+1. The mincount for /Core/ElementCollection/element is 1
+1. The mincount for /Core/ElementCollection/rootElement is 1
+1. The mincount for /Core/Element/spdxId is 1
+1. The mincount for /Core/Element/creationInfo is 1
+
+For a /Core/Element/CreationInfo to be conformant with this profile,
+the following has to hold:
```

Comment:
All of these are not needed:
- `specVersion`, `created`, and `createdBy` have already mincount 1 in Core
- `profile` was moved (I think)
- `dataLicense` is... yet to be determined, but it most probably won't be there